pages/linux/waypipe.md

```diff
@@ -0,0 +1,12 @@
+# waypipe
+
+> Remotely run graphical applications under a Wayland compositor.
+> More information: <https://gitlab.freedesktop.org/mstoeckl/waypipe>.
+
+- Run a graphical program remotely and display it locally:
+
+`waypipe ssh {{user}}@{{server}} {{program}}`
+
+- Open an ssh tunnel to run any program remotely and display it locally:
```

Comment:
```suggestion
- Open a ssh tunnel to run any program remotely and display it locally:
```

Reply:
It's pronounced "ess-ess-aitch", thus "an" is correct